test/users/npadmana/fftw/testFFTW.chpl

```diff
@@ -102,7 +175,13 @@ proc runtest(param ndim : int, fn : string) {
 	destroy_plan(fwd);
 	destroy_plan(rev);
 
-  // Testing r2c and c2r
+  /* Real <-> complex out-of-place transform 
+
+     As with FFTW, these use an r2c and c2r suffix to define the direction of the transform.
+
+     plan_dft_r2c/plan_dft_c2r are overloaded; for the out-of-place transforms, they infer the
+     dimensions from the sizes of the arrays passed in.
+  */
   var rA : [D] real(64); // No padding for an out-of place transform
   var cB : [cD] complex;
```

Comment:
Minor point: This is the only place I can think of in the FFT modules in which complex is used rather than complex(128).